fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/session/HttpSession.java

```diff
@@ -27,32 +27,25 @@
 
     private boolean batch = false;
 
-    private final FedoraSession session;
+    private final FedoraTransaction transaction;
 
     /**
-     * Create an HTTP session from a Fedora session
-     * @param session the Fedora session
+     * Create an HTTP session from a Fedora transaction
+     * @param transaction the Fedora transaction
      * Note: by default, the HTTP Session is not marked as a batch operation.
      * Client code must call makeBatch in order to promote the session into
      * a batch operation.
      */
-    public HttpSession(final FedoraSession session) {
-        this.session = session;
-    }
-
-    /**
-     * Make this HTTP Session into a batch operation.
-     */
-    public void makeBatchSession() {
-        this.batch = true;
+    public HttpSession(final FedoraTransaction transaction) {
```

Comment:
What distinct functionality is HttpSession adding that the FedoraTransaction doesn't provide? When I was looking through the FedoraSession and HttpSession, I mainly had the impression that HttpSession added in the ability to have multi-request sessions. That idea seems to be implied by FedoraTransaction now, so long as isShortLived isn't true.

Reply:
HttpSession would track the request level state of the transaction.    That information should not be associated with the transaction,  I believe.   So one transaction, two simultaneous requests, one request has a "Atomic-Commit", the other doesn't.

Reply:
Decision from meeting: HttpSession can be deleted and relevant methods (UpdateExpiry, getExpires) can be moved to FedoraTransaction interface. 
As per the transaction sidecar PR, transaction commit happens only through a separate request to the transactions endpoint. Using "Atomic-Commit" header to commit a transaction will not be supported.